api-specificatie/openapi.yaml

```diff
@@ -1130,32 +1165,25 @@ components:
       type: object
       properties:
         gebruiksdoel:
-          description: Indicator die aangeeft of er onderzoek wordt gedaan naar de
-            juistheid van de waarde in het corresponderende gegeven.
+          description: "Indicator die aangeeft of er onderzoek wordt gedaan naar de juistheid van de waarde in het corresponderende gegeven."
           type: boolean
         geometrie:
-          description: Indicator die aangeeft of er onderzoek wordt gedaan naar de
-            juistheid van de waarde in het corresponderende gegeven.
+          description: "Indicator die aangeeft of er onderzoek wordt gedaan naar de juistheid van de waarde in het corresponderende gegeven."
           type: boolean
         hoofdadresidentificatie:
-          description: Indicator die aangeeft of er onderzoek wordt gedaan naar de
-            juistheid van de waarde in het corresponderende gegeven.
+          description: "Indicator die aangeeft of er onderzoek wordt gedaan naar de juistheid van de waarde in het corresponderende gegeven."
           type: boolean
         nevenadresIdentificaties:
-          description: Indicator die aangeeft of er onderzoek wordt gedaan naar de
-            juistheid van de waarde in het corresponderende gegeven.
+          description: "Indicator die aangeeft of er onderzoek wordt gedaan naar de juistheid van de waarde in het corresponderende gegeven."
           type: boolean
         pandIdentificaties:
-          description: Indicator die aangeeft of er onderzoek wordt gedaan naar de
-            juistheid van de waarde in het corresponderende gegeven.
+          description: "Indicator die aangeeft of er onderzoek wordt gedaan naar de juistheid van de waarde in het corresponderende gegeven."
           type: boolean
         oppervlakte:
-          description: Indicator die aangeeft of er onderzoek wordt gedaan naar de
-            juistheid van de waarde in het corresponderende gegeven.
+          description: "Indicator die aangeeft of er onderzoek wordt gedaan naar de juistheid van de waarde in het corresponderende gegeven."
           type: boolean
         status:
-          description: Indicator die aangeeft of er onderzoek wordt gedaan naar de
-            juistheid van de waarde in het corresponderende gegeven.
+          description: "Indicator die aangeeft of er onderzoek wordt gedaan naar de juistheid van de waarde in het corresponderende gegeven."
           type: boolean
     AdresseerbaarObject_embedded:
       type: object
```

Comment:
adres moet zijn hoofdAdres, want zo heet het ook in _links

Reply:
Aangepast